File: Feitos em C++ Parte 2/1541.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
	int x;
	while(true){
		int a,b,c,d;
		cin>>a;
		if(a==0){
			break;
		}
		cin>>b>>c;
		int resp = sqrt((a*b*100)/c);
		cout<<resp<<endl;
	}
	return 0;
}

File: Feitos em Java/1541.java
import java.io.IOException;
import static java.lang.Math.sqrt;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner (System.in);
        int x=0;
    	while(true){
    		int a=entrada.nextInt();
    		if(a==0){
    			break;
    		}
    		int b=entrada.nextInt();
                    int c=entrada.nextInt();
    		int resp = (int) sqrt((a*b*100)/c);
    		System.out.println(resp);
    	}
    }
}